.github/workflows/check.yaml

```diff
@@ -10,9 +10,11 @@ on:
       - ready_for_review
     branches:
       - main
+      - 503-introduce-shinytest2@main
   push:
     branches:
       - main
+      - 503-introduce-shinytest2@main
```

Comment:
This must be reverted before merging to main

Reply:
reverted